modules/custom/az_block_types/az_custom_menu_block/az_custom_menu_block.services.yml

```
@@ -0,0 +1,6 @@
+services:
+  az_custom_menu_block.az_kernel_view_subscriber:
```

Nitpick: I don't think the `az_` prefix is needed since the service is already prefixed with the module name.
```suggestion
  az_custom_menu_block.kernel_view_subscriber:
```